websites/A/AnimeHeaven/presence.ts

```diff
@@ -0,0 +1,161 @@
+const presence = new Presence({
+    clientId: "816042675626442783"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+  });
+    let iFrameVideo: boolean,
+    currentTime: number,
+    duration: number,
+    paused: boolean,
+    video: {
+        iframe_video: {
+        duration: number;
+        iFrameVideo: boolean;
+        currTime: number;
+        dur: number;
+        paused: boolean; 
+        }
+        }, playback: boolean, search: string;
+
+    presence.on(
+            "iFrameData",
+            (data: {
+            iframe_video: {
+                duration: number;
+                iFrameVideo: boolean;
+                currTime: number;
+                dur: number;
+                paused: boolean;
+            };
+            }) => {
+            playback = data.iframe_video.duration !== null ? true : false;
+            if (playback) {
+                iFrameVideo = data.iframe_video.iFrameVideo;
+                currentTime = data.iframe_video.currTime;
+                duration = data.iframe_video.dur;
+                paused = data.iframe_video.paused;
+                video = data;
+            }
+            }
+        );
+
+    const browsingStamp = Math.floor(Date.now() / 1000), path = document.location.pathname;
+    let title, link;
+    presence.on("UpdateData", async () => {
+        const buttons = await presence.getSetting("buttons"), videoTime = await presence.getSetting("sVT"),
+        data: PresenceData = {
+        largeImageKey: "logo"
+        }, timestamps = presence.getTimestamps(
+            Math.floor(currentTime),
+            Math.floor(duration)
+        );
+
+        if(path == "/animeheaven.eu" || path == "/"){
+            data.details = "Viewing Homepage";
+            data.startTimestamp = browsingStamp;
```

Comment:
is it necessary to assign it everytime? :/ just assign it to ``data`` object please.

Reply:
This does work. Scared it wasn't going to because of the video timestamps but because it starts off by deleting them it is good.